src/xcode/ENA/ENA/Source/View Helpers/SeperatorLineLayer.swift

```diff
@@ -0,0 +1,18 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class SeperatorLineLayer: CAShapeLayer {
+	
+	override init() {
+		super.init()
+		lineWidth = 1
+		strokeColor = UIColor.enaColor(for: .hairline).cgColor
+	}
+	
+	required init?(coder: NSCoder) {
```

Comment:
Putting "@available(*, unavailable)" would be a nice thing for the compiler, our beloved friend.